react-components/package.json

```diff
@@ -104,7 +104,7 @@
     "three": "0.166.1",
     "ts-loader": "^9.5.1",
     "typescript": "^5.4.5",
-    "vite": "^5.4.15",
+    "vite": "^6.0.0",
```

Comment:
There are newer patch versions. Upgrading to a x.0.0 when there are newer patches for that minor should be avoided.

This is a major version upgrade, so we should be careful. Have you gone through the release notes?

Reply:
No, I have not - I decided that this would be safe to release since it only affects the build steps. But I understand that we should be careful. I will look through the release notes when I have the time

Reply:
@psalaberria002 The release notes and migration guide did note contain any unwelcome surprises. I also upgraded to the newest minor/patch versions. Ready for another review :pray: